frontend/src/UpdateBook.tsx

```diff
@@ -96,13 +97,12 @@ function UpdateBook(props: UpdateBookProps) {
                     <option value={BookState.NOT_AVAILABLE}>NOT_AVAILABLE</option>
                 </select>
                 <br/><br/>
-                <button>Update</button>
+                <div className="modal-body">
+                    <h5>Are you sure to update this book?</h5>
```

Comment:
Semantically, in order to have a h5, there have to be h1-h4 before that.
Consider using a different heading or a \<p> with styling